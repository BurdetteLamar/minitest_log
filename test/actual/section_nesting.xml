<log>
  <summary verdicts='0' failures='0' errors='0'/>
  <section name='no_nesting'/>
  <section name='outer'>
    <section name='inner'/>
  </section>
  <section name='outer'>
    <section name='mid'>
      <section name='inner'/>
    </section>
  </section>
</log>
