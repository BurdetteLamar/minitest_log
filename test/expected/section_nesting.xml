<log>
  <section_ name='no_nesting'/>
  <section_ name='outer'>
    <section_ name='inner'/>
  </section_>
  <section_ name='outer'>
    <section_ name='mid'>
      <section_ name='inner'/>
    </section_>
  </section_>
</log>
